Source/WebCore/css/CSSStyleSheet.h

```diff
@@ -139,6 +161,9 @@ class CSSStyleSheet final : public StyleSheet {
     CSSStyleSheet(Ref<StyleSheetContents>&&, CSSImportRule* ownerRule);
     CSSStyleSheet(Ref<StyleSheetContents>&&, Node* ownerNode, const TextPosition& startPosition, bool isInlineStylesheet);
     CSSStyleSheet(Ref<StyleSheetContents>&&, Node& ownerNode, const TextPosition& startPosition, bool isInlineStylesheet, const std::optional<bool>&);
+    CSSStyleSheet(Ref<StyleSheetContents>&&, Document&, Init&&);
+
+    void forEachStyleScope(const Function<void(Style::Scope&)>&);
```

Comment:
I think this is only ever used internally in this class? It would be better not to expose it as public. Also it could take template argument instead of Function and perhaps even be a standalone helper in cpp?

Reply:
It is already private. Making it a standalone in the cpp is a bit annoying since it relies on CSSStyleSheet data members.